core/modules/proto/worker.proto

```diff
@@ -52,7 +52,14 @@ message TaskMsg {
         // contributing fragments' rows.
     }
     repeated Fragment fragment = 5;
-
+    message ScanTable {
+        required string db = 1;
+        required string table = 2;
+        required bool lockInMemory = 3;
+        required int32 scanSpeed = 4;
+    }
+    repeated ScanTable scantable = 4;
```

Comment:
Easy enough to change and I don't have much of an opinion on this. Our protocol case is a bit unusual in that it's purely internal, and the czar and worker are built at the same time. string scantables = 4 will never be used again, not much point in leaving a reference to it.